cranelift/src/disasm.rs

```diff
@@ -118,6 +118,11 @@ cfg_if! {
                     .mode(arch::sysz::ArchMode::Default)
                     .build()
                     .map_err(map_caperr)?,
+                Architecture::Riscv64 {..} => Capstone::new()
+                    .riscv()
+                    .mode(arch::riscv::ArchMode::RiscV64)
+                    .build()
+                    .map_err(map_caperr)?,
```

Comment:
Probably needs `cs.set_skipdata(true)` just like aarch64 to handle inline constants.